self-core-impl/src/test/java/com/selfxdsd/core/mock/InMemoryProjectsTestCase.java

```diff
@@ -20,6 +20,8 @@
  * @author hpetrila
  * @version $Id$
  * @since 0.0.1
+ * @todo #96:30min Continue tests for {@link Projects#getProjectById(int)}
+ *  in PmProjectsTestCase and UserProjectsTestCase.
```

Comment:
@BogdanAdrian11 Here, please update this puzzle to specify that each implementation of ``Projects`` should provide their own ``getById`` method. 

For instance, in the case of ``UserProjects``, the method will be allowed to use ``iterator()`` since we can afford to load all of a User's projects in memory and iterate them.